src/main/java/org/zalando/nakadi/service/timeline/TimelineService.java

```diff
@@ -356,6 +360,7 @@ private void finishTimelineUpdate(final String eventTypeName) throws TimelineExc
 
         try {
             final EventType eventType = eventTypeCache.getEventType(eventTypeName);
+            authorizationValidator.authorizeEventTypeView(eventType);
```

Comment:
and get-timelines as well. It's allowed only to Nakadi admins, so I think it doesn't make sense to check if event-type-view is allowed.